process_pr.py

```diff
@@ -2393,7 +2398,7 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
 
     logger.debug("All Parameters: %s", global_test_params)
     # For now, only trigger tests for cms-sw/cmssw and cms-sw/cmsdist
-    if create_test_property:
+    if create_test_property and bot_status is not None and not is_closed_pr:
```

Comment:
@iarspider , checking for `bot_status` has no use here. It might be empty for first run but on next bot run it will have some statuses . all of your current changes are not needed.  A simple change like 
```
if pr.state == "closed":
  create_test_property=False
``` 
after https://github.com/cms-sw/cms-bot/blob/master/process_pr.py#L1080 line will stop running PR tests for closed PRs. 

I also wanted to avoid creating new statuses if PR is closed and old statuses are empty. It could be that there is bug on github side and they restore the old commit statuses at some point. If we create new statuses then they might not be able to restore them

Reply:
Ah, I see. I will now work on the second part of this PR (not updating statuses).